frappe/templates/emails/auto_email_report.html

```diff
@@ -39,7 +39,11 @@
 					</td>
 				{% else %}
 					<td {{- get_alignment(col) }}>
-						{{- frappe.format(row[col.fieldname], col, row) -}}
+						{% if col.fieldtype in ("Link", "Dynamic Link") and col.options %}
```

Comment:
@Alchez I checked and found that hyperlinks show up for "Report Builder" reports as well other than the name field that is primary key of the document, for which you have already made the changes in report.py. No need to make any changes in auto_email_report.html that part is already handled in auto_email_report. Also the changes you have made here won't work for Dynamic links since the options for a Dynamic Link is  a fieldname and not a Doctype.

Reply:
Ah, makes sense. I'll remove changes from this file. Can I ask you to test this again after please?